### DISCUSSION TITLE
PCL2的版本号到了2.9.x那下一个大版本会是什么版本号

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3614] [提问] [answered]
\

**Comment by iPodtouch6557** [comment, marked as answer]
2.10.x

**Comment by wuliaodexiaoluo** [comment]
也可能是 PCL 3.x ( 3.0 时代) 也说不定

**Reply by MoYuan-CN** [reply]
不会的吧....

**Reply by wuliaodexiaoluo** [reply]
事情还没发生的说，除非龙猫表明没有 PCL 3，不然不会存在这种说法

**Reply by MoYuan-CN** [reply]
3.0 应该没有这么快来

**Reply by wuliaodexiaoluo** [reply]
可能还没到 3.0 Minecraft 就凉掉了。。。。

**Reply by iPodtouch6557** [reply]
不至于吧

**Reply by Deep-Dark-Forest** [reply]
> 事情还没发生的说，除非龙猫表明没有 PCL 3，不然不会存在这种说法

回声洞里面明确表示：啥时候出 PCL3？回：不可能啦
我没法确定这是不是龙猫回的……

**Reply by hejiehao** [reply]
~~https://github.com/PCL-Community/PCL3~~
![79e498bc3b928541dea96ec34a514867](https://github.com/Hex-Dragon/PCL2/assets/98140773/fab7d3db-9bac-442e-b0fd-6c137a66d2c5)
![8fb0f72be9d67bdbf69b597779256b32](https://github.com/Hex-Dragon/PCL2/assets/98140773/b2e2275c-0e56-48c4-8c7b-6a3b42d5a5cf)

**Reply by MoYuan-CN** [reply]
~~谁™在组织里自 Fork 了一下改了名~~

**Reply by Deep-Dark-Forest** [reply]
> ~~谁™在组织里自 Fork 了一下改了名~~

我当时创建fork的时候起的名就是PCL3，没想到创建完删不掉。。。

**Reply by ZerkyLiu** [reply]
> > 事情还没发生的说，除非龙猫表明没有 PCL 3，不然不会存在这种说法
> 
> 回声洞里面明确表示：啥时候出 PCL3？回：不可能啦 我没法确定这是不是龙猫回的……

在哪里可以看回声洞的所有留言？PCL里一个一个点太麻烦了

**Reply by MoYuan-CN** [reply]
> 在哪里可以看回声洞的所有留言？PCL里一个一个点太麻烦了

好像没有开放渠道

**Reply by zkitefly** [reply]
> > > 事情还没发生的说，除非龙猫表明没有 PCL 3，不然不会存在这种说法
> > 
> > 
> > 回声洞里面明确表示：啥时候出 PCL3？回：不可能啦 我没法确定这是不是龙猫回的……
> 
> 在哪里可以看回声洞的所有留言？PCL里一个一个点太麻烦了

https://github.com/Hex-Dragon/PCL2/blob/56cb6bc7d9ab96801c1357f9516bd796f48f0f97/Plain%20Craft%20Launcher%202/Pages/PageOther/PageOtherTest.xaml.vb#L17-L18

**Reply by Deep-Dark-Forest** [reply]
> > > 事情还没发生的说，除非龙猫表明没有 PCL 3，不然不会存在这种说法
> > 
> > 回声洞里面明确表示：啥时候出 PCL3？回：不可能啦 我没法确定这是不是龙猫回的……
> 
> 在哪里可以看回声洞的所有留言？PCL里一个一个点太麻烦了

你可以把动画速度调到最大然后一直点就行了

**Comment by ZS2147483647** [comment]
PCL 2 Gen2*1
PCL 2 Gen2*2
PCL 2 Gen2*3
......

**Reply by Pigeon0v0** [reply]
什么 USB Forum
